service/eventer/projects.go

```diff
@@ -0,0 +1,32 @@
+package eventer
+
+var (
+	commonProjectList = []string{
+		"api",
+		"cluster-operator",
+		"credentiald",
+		"draughsman",
+		"happa",
+		"passage",
+		"vault-exporter",
+	}
+	awsProjectList = []string{
+		"aws-app-collection",
+		"aws-operator",
+	}
+	kvmProjectList = []string{
+		"kvm-app-collection",
+		"kvm-operator",
+		"ingress-operator",
```

Comment:
Is this a thing anymore? I think it was merged into `kvm-operator`. https://github.com/giantswarm/ingress-operator is archived :D

Reply:
Wow, a lot of dead repositories :D